src/backoffice/shared/components/workspace/workspace-property-layout/workspace-property-layout.element.ts

```diff
@@ -21,6 +21,11 @@ export class UmbWorkspacePropertyLayoutElement extends LitElement {
 				border-bottom: 1px solid var(--uui-color-divider);
 				padding: var(--uui-size-space-6) 0;
 				container-type: inline-size;
+				position: relative;
+			}
+
+			:host([alias='eyeDropper']) {
+				z-index: 2;
```

Comment:
Hi @nielslyngsoe, yes. I think an option is to make z-indexes order like in Boostrap library. 

`$zindex-dropdown:                   1000;`
`$zindex-sticky:                     1020;`
`$zindex-fixed:                      1030;`
`$zindex-modal-backdrop:             1040;`

etc. I can create a few z-indexes classes and use these classes in code. But still, I need based on alias add class for this specific `eyeDropper` component.

Reply:
I see one more option, add bigger z-index to `uui-popover` from [uui.umbraco](https://uui.umbraco.com/?path=/docs/uui-popover--aaa-overview), but it may touch more connected reports.
Unfortunately, I don't see other alternatives than position relative and z-index. But it's CSS, maybe there is a solution, but in the current state of my styling knowledge, I don't see it.

Reply:
Cool, and thanks for your enthusiasm!

An idea could be to be able to control the popover z-index via a CSS Custom Property. In this way we could control the actual number within Backoffice.

I'll create an issue on UUI.

Reply:
I changed my mind and made a PR instead. So let's return to this one when that's merged. Thanks again.
https://github.com/umbraco/Umbraco.UI/pull/441